pyproject.toml

```
@@ -59,6 +59,7 @@ python-jose = { version = "^3.3.0", extras = ["cryptography"], optional = true }
 jinja2 = { version = "^3.1.4", optional = true }
 python-multipart = { version = ">=0.0.9,<0.0.18", optional = true }
 celery = { version = "^5.4.0", optional = true }
+pyspark = { version = ">=3.5.1,<3.5.2", optional = true }
 onetl = { version = "^0.12.0", extras = ["spark"], optional = true }
```

as I see from [onetl docs](https://onetl.readthedocs.io/en/stable/file_df/file_formats/excel.html) it compatible with 3.2.x - 3.5.x versions
also if we add pyspark as dependency, we do not need extras spark in onetl anymore as it is installed explicitly 

```suggestion
pyspark = { version = ">=3.5.1,<3.5.2", optional = true }
onetl = { version = "^0.12.0", optional = true }
```

---

It simply doesn't work with 3.2+ versions, also found [this source](https://mvnrepository.com/artifact/com.crealytics/spark-excel) with 3.5.1

---

Another way is to call `Excel.get_packages(spark_version="3.5.1")`. It should support all patch versions, but Spark version is in the name of Maven package, so we have to set it somehow